doc/release/upcoming_changes/21976.new_feature.rst

```diff
@@ -0,0 +1,31 @@
+The bit generator underlying the singleton RandomState can be changed
+---------------------------------------------------------------------
+The singleton ``RandomState`` instance exposed in the ``numpy.random`` module
+id initialized using system-provided entropy with the ``MT19937` bit generator.
+The function ``set_bit_generator`` allows the default bit generator to be
+replaced with a user-provided bit generator. This function has been introduced to
+provide a method allowing seemless integration of a high-quality, modern bit generator
+in new code with existing code that makes use of the singleton-provided random
+variate generating functions. The companion function ``get_bit_generator`` returns the current bit generator
+being used by the singleton ``RandomState``. This is provided to simplify restoring
+the original source of randomness if required.
```

Comment:
Could you reflow the paragraph, it goes way over 80 columns

Reply:
Done.